core/server/data/schema/schema.js

```diff
@@ -308,5 +308,40 @@ module.exports = {
         session_data: {type: 'string', maxlength: 2000, nullable: false},
         created_at: {type: 'dateTime', nullable: false},
         updated_at: {type: 'dateTime', nullable: true}
+    },
+    integrations: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        name: {type: 'string', maxlength: 191, nullable: false},
+        slug: {type: 'string', maxlength: 191, nullable: false, unique: true},
+        icon_image: {type: 'string', maxlength: 2000, nullable: true},
+        description: {type: 'string', maxlength: 2000, nullable: true},
+        created_at: {type: 'dateTime', nullable: false},
+        created_by: {type: 'string', maxlength: 24, nullable: false},
+        updated_at: {type: 'dateTime', nullable: true},
+        updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    api_keys: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        type: {
+            type: 'string',
+            maxlength: 50,
+            nullable: false,
+            validations: {isIn: [['content', 'admin']]}
+        },
+        secret: {
+            type: 'string',
+            maxlength: 191,
+            nullable: false,
+            unique: true,
+            validations: {isLength: {min: 128, max: 128}}
+        },
+        role_id: {type: 'string', maxlength: 24, nullable: true},
+        integration_id: {type: 'string', maxlength: 24, nullable: true},
```

Comment:
Curious why this is nullable.

Reply:
An API Key with no associated integration is considered "internal" and won't show up in the UI. Useful for clients such as the scheduler or Ghost(pro) backup client that need API access but that would be confusing if shown in the UI.

Reply:
Cool 👍 Can we add a comment somewhere? Thanks!